docs/assets/js/src/application.js

```diff
@@ -36,50 +36,38 @@
       e.preventDefault()
     })
 
-    // Config ZeroClipboard
-    ZeroClipboard.config({
-      moviePath: '/assets/flash/ZeroClipboard.swf',
-      hoverClass: 'btn-clipboard-hover'
-    })
-
     // Insert copy to clipboard button before .highlight
     $('.highlight').each(function () {
-      var btnHtml = '<div class="zero-clipboard"><span class="btn-clipboard">Copy</span></div>'
+      var btnHtml = '<div class="bd-clipboard"><span class="btn-clipboard" title="Copy to clipboard">Copy</span></div>'
       $(this).before(btnHtml)
-    })
-    var zeroClipboard = new ZeroClipboard($('.btn-clipboard'))
-    var $htmlBridge = $('#global-zeroclipboard-html-bridge')
-
-    // Handlers for ZeroClipboard
-    zeroClipboard.on('load', function () {
-      $htmlBridge
-        .data('placement', 'top')
-        .attr('title', 'Copy to clipboard')
-        .tooltip()
-
-      // Copy to clipboard
-      zeroClipboard.on('dataRequested', function (client) {
-        var highlight = $(this).parent().nextAll('.highlight').first()
-        client.setText(highlight.text())
-      })
-
-      // Notify copy success and reset tooltip title
-      zeroClipboard.on('complete', function () {
-        $htmlBridge
-          .attr('title', 'Copied!')
-          .tooltip('fixTitle')
-          .tooltip('show')
-          .attr('title', 'Copy to clipboard')
-          .tooltip('fixTitle')
-      })
+      $('.btn-clipboard').tooltip()
     })
 
-    // Hide copy button when no Flash is found
-    // or wrong Flash version is present
-    zeroClipboard.on('noflash wrongflash', function () {
-      $('.zero-clipboard').remove()
-      ZeroClipboard.destroy()
-    })
+    var clipboard = new Clipboard('.btn-clipboard', {
+      target: function (trigger) {
+        return trigger.parentNode.nextElementSibling
+      }
+    });
+
+    clipboard.on('success', function (e) {
+      $(e.trigger)
+        .tooltip('dispose')
+        .attr('title', 'Copied!')
+        .tooltip('show')
+        .attr('data-original-title', 'Copy to clipboard')
+
+      e.clearSelection()
+    });
+
+    clipboard.on('error', function (e) {
+      var fallbackMsg = /Mac/i.test(navigator.userAgent) ? 'Press ⌘-C to copy' : 'Press Ctrl-C to copy'
```

Comment:
I agree. I've been meaning to do that but it got late. I'll get to it
tomorrow.
On Oct 14, 2015 01:32, "Chris Rebert" <notifications@github.com> wrote:

> In docs/assets/js/src/application.js
> <https://github.com/twbs/bootstrap/pull/17751#discussion_r41934913>:
>
> > +        return trigger.parentNode.nextElementSibling
> > +      }
> > +    });
> > +
> > +    clipboard.on('success', function (e) {
> > +      $(e.trigger)
> > +        .tooltip('dispose')
> > +        .attr('title', 'Copied!')
> > +        .tooltip('show')
> > +        .attr('data-original-title', 'Copy to clipboard')
> > +
> > +      e.clearSelection()
> > +    });
> > +
> > +    clipboard.on('error', function (e) {
> > +      var fallbackMsg = /Mac/i.test(navigator.userAgent) ? 'Press ⌘-C to copy' : 'Press Ctrl-C to copy'
>
> Perhaps we could use \u2318 instead of a literal ⌘ so as to keep this
> file in ASCII?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twbs/bootstrap/pull/17751/files#r41934913>.
>